providers/src/airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -362,11 +362,11 @@ def __init__(self, *args: Any, virtual_cluster_id: str | None = None, **kwargs:
     # Retry this method when the ``create_virtual_cluster`` raises
     # "Cluster XXX is not reachable as its connection is currently being updated".
     # Even though the EKS cluster status is ``ACTIVE``, ``create_virtual_cluster`` can raise this error.
-    # Retrying is the only option.
+    # Retrying is the only option. Retry up to 3 minutes
```

Comment:
This is starting to be quite a long wait. That's a pretty big regression considering we haven't needed any wait previously?

Reply:
Agree :) But I dont have any other solution either. And this is something users can feel the pain too, so I think it is better to wait than to fail